src/design-system/InputText.tsx

```diff
@@ -16,20 +16,36 @@ const VDiv = styled.div`
 
 interface Props extends InputBaseProps<string> {
   type: "text" | "number";
+  headerStyle?: boolean;
+  onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void;
+  onKeyDown?: (event: React.KeyboardEvent) => void;
+  focus?: boolean;
 }
 
 const InputText: React.FC<Props> = (props) => {
   let inputValue = useInputValue(props);
 
+  const nameInput = useRef() as React.MutableRefObject<HTMLInputElement>;
```

Comment:
@itsjon It's kind of hard to discover, but the way to do this without casting is

```suggestion
  const nameInput = useRef<HTMLInputElement>(null);
```